src/bucketd/processor.rs

```diff
@@ -193,6 +199,37 @@ impl Runtime {
             self.store.store_sten(db::SCHEMATA, root_schema.schema_id(), root_schema)?;
         }
 
+        match reveal {
+            Some(Reveal {
+                blinding_factor,
+                outpoint,
+                close_method,
+            }) => {
```

Comment:
This is usually done by using `matches!` macro, which avoids the last `_ => none!()` thing. Usually this is reported by the linter - no idea why it passed here, probably linting CI got broken